webapp/sources/rudder/rudder-core/src/test/resources/configuration-repository/expected-share/node-cfe-with-500-directives/rules/cfengine-community/rudder.json

```diff
@@ -3,11 +3,7 @@
   "AGENT_RUN_SCHEDULE":"\"Min01\", \"Min06\", \"Min11\", \"Min16\", \"Min21\", \"Min26\", \"Min31\", \"Min36\", \"Min41\", \"Min46\", \"Min51\", \"Min56\"",
   "AGENT_RUN_SPLAYTIME":"4",
   "AGENT_TYPE":"cfengine-community",
-  "AUTHORIZED_NETWORKS":[
-    "192.168.12.0/24",
-    "192.168.49.0/24",
-    "127.0.0.1/24"
-  ],
+  "AUTHORIZED_NETWORKS":[],
```

Comment:
Do we keep `AUTHORIZED_NETWORKS` instead of `ALLOWED_NETWOKS` ?

Reply:
We decided to change